### DISCUSSION TITLE
Odds n' ends

### DISCUSSION THREAD

**Discussion by biggnome** [avaraline/Avara#154] [Show and tell]
I have a few little curiosities on a virtual drive from the old days of Avara I'd be happy to contribute if there's any interest, namely the Avara Collection 1 level set from one of the old Ambrosia Times newsletters, and a fan mod featuring a few high(-er) resolution models and some alternative sound effects which was probably posted on the Avara boards at some point. I also have my own little LaTeX reconstruction of the original documentation I put together recently as kind of a "hello world" project.

https://github.com/biggnome/Avara-stuff

**Comment by assertivist** [comment]
Amazing!!

I looked and it seems I we dont have Avara Collection in my archive (thought I definitely remember it), so thank you for that!

if you happen to have OSX, you can make the binary files that the port can load yourself with [this script](https://github.com/avaraline/Avara/blob/master/bin/levelset2rsrc) included in the `bin` directory of the source

If you post those I can run [rsrc2files](https://github.com/avaraline/Avara/blob/master/bin/rsrc2files.py) (it has quite a few python dependencies, but you're welcome to try installing them and running it yourself too) to get the BSPs out of the mod. THEN, finally, you can drag those into the `bsps` folder of the game and see them. I have a [blender exporter](https://github.com/assertivist/avarabsp-blender-exporter) for the format but I need to put together the importer still.

But also if you wait long enough I will eventually get around to borrowing a mac for a few minutes to do it myself :)

**Comment by biggnome** [comment]
Janky OS 10.6 VM to the rescue--.r files attached (I found a neat old exploration-focused solo level as well you may find interesting). Did my best to extract BSPS, but my computer was just not having it unfortunately, and I'm pretty out of my element with python.
[c1-resources-misc.zip](https://github.com/avaraline/Avara/files/5957129/c1-resources-misc.zip)

I did semi-successfully extract some audio resources from the above while I was at it, and I noticed what I appears to be an issue, although I couldn't be certain whether it's an error on my machine, or an idiosyncrasy that plays well with Avara's sound engine or what. I'll make a detailed post about it in the issue tracker.

**Reply by assertivist** [reply]
Nice, those worked! Man, the levels list is starting to get too long. 
![2021-02-10-231553_2560x1080_scrot](https://user-images.githubusercontent.com/1486313/107610367-4a715400-6bf6-11eb-9ffb-ad40d97d1d9e.png)
Gross!! :laughing: 
I don't remember ever seeing this. Looks like there's some front/back face issues, thats something I need to fix in the triangulator thing.
I'll chuck those new sets in tomorrow.
